dbms/src/Storages/DeltaMerge/Delta/DeltaValueSpace.h

```diff
@@ -94,7 +94,10 @@ class DeltaValueSpace
     std::atomic<size_t> last_try_place_delta_index_rows = 0;
 
     DeltaIndexPtr delta_index;
-    UInt64 delta_index_epoch = 0;
+    // `delta_index_epoch` is used in disaggregated mode by compute-nodes to identify if the delta_index has changed.
+    // To avoid persisting it, we use `std::chrono::steady_clock` to make it monotonic increase.
+    // Accuracy of `std::chrono::steady_clock` in Linux is nanosecond. This is much smaller than the interval between two flushes.
+    UInt64 delta_index_epoch = std::chrono::steady_clock::now().time_since_epoch().count();
```

Comment:
We should also update delta_index_epoch here to be `std::chrono::steady_clock::now().time_since_epoch().count()`

https://github.com/pingcap/tiflash/blob/2b3b2f5fe302a655abf160ec7b4585939f9ded92/dbms/src/Storages/DeltaMerge/Delta/DeltaValueSpace.cpp#L385-L393